Mathlib/Data/Equiv/Basic.lean

```diff
@@ -43,12 +43,20 @@ rfl
 
 def refl (α) : α ≃ α := ⟨id, id, λ _ => rfl, λ _ => rfl⟩
 
+instance : Inhabited (α ≃ α) := ⟨equiv.refl α⟩
+
 def symm (e : α ≃ β) : β ≃ α := ⟨e.inv_fun, e.to_fun, e.right_inv, e.left_inv⟩
 
+def simps.symm_apply (e : α ≃ β) : β → α := e.symm
+
 def trans (e₁ : α ≃ β) (e₂ : β ≃ γ) : α ≃ γ :=
 ⟨e₂ ∘ (e₁ : α → β), e₁.symm ∘ (e₂.symm : γ → β),
   e₂.left_inv.comp e₁.left_inv, e₂.right_inv.comp e₁.right_inv⟩
 
+@[simp] theorem to_fun_as_coe (e : α ≃ β) : e.to_fun = e := rfl
```

Comment:
BTW, please check the linter output.  It says this theorem shouldn't be marked simp (if you hover over rfl you might see why).